exec/cnex/object.h

```diff
@@ -45,6 +46,7 @@ Object *object_createStringObject(TString *s);
 Object *object_createFileObject(FILE *f);
 Object *object_createProcessObject(void *h);
 Object *object_createMMapObject(struct MmapObject *m);
+Object *object_createDatabaseObject(struct sqlite3 *db);
```

Comment:
There's no need to have this declaration here, there does not need to be any changes to `object.h` at all.

Reply:
Removed.